config/settings/test.yml

```diff
@@ -225,6 +225,12 @@ lighthouse:
         client_id: abc123456
         rsa_key: path/to/key
       use_mocks: false
+  benefits_education:
+    host: https://sandbox-api.va.gov
+    use_mocks: true
+    access_token:
+      client_id: abc123456
+      rsa_key: path/to/key
```

Comment:
In order to execute the code in a test (and have it hit the external api):
- make `use_mocks` false 
- set client_id and path to `private.pem` file on your local machine

Reply:
Love these notes! What do you think about adding them to our Google Drive or somewhere else so that the team has access to them?

Reply:
Thanks.  Yeah I included the notes in comments in a test, but it would be nice to have them stored in a more formal document.  Maybe they could be added to the [vcr debugging](urhttps://depo-platform-documentation.scrollhelp.site/developer-docs/vcr-debuggingl) notes in the platform documentation.  Since this is my first opportunity to use VCR, my advice might be too overly narrow, fixating on the specific task at hand.

That said, documentation should never be considered static or "complete".  I will make a submission to the platform documentation via [this form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=&labels=documentation-support&template=documentation-support-request.yml&title=Documentation+support)

Reply:
@AdamKing0126 You can address the unknown path like this:  `rsa_key: ~`